src/components/file-dropzone/file-dropzone.scss

```diff
@@ -0,0 +1,46 @@
+:host(limel-file-dropzone) {
+    position: relative;
+    display: grid;
+    position: relative;
+    height: 100%;
+    width: 100%;
+}
+
+.has-file-to-drop {
+    position: absolute;
+    top: 0;
+    left: 0;
+    overflow: hidden;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 100%;
+    box-sizing: border-box;
+
+    border: 0.25rem dashed rgb(var(--color-gray-light));
+    border-radius: 0.5rem;
+    background-color: rgb(var(--color-gray-dark));
+
+    opacity: 0.9;
+    isolation: isolate;
+    z-index: 1;
+
+    limel-icon.icon {
+        height: clamp(2rem, 4vh, 6rem);
+        width: clamp(2rem, 4vh, 6rem);
+        color: rgb(var(--contrast-800));
+    }
+
+    .text {
+        font-size: clamp(1rem, 2vh, 1.75rem);
```

Comment:
🥰 I love it that you did this! 😇

Reply:
But I think these sizes are still too big when the container is small. Let's say we use this on `limel-file`, will the text be visible? 🤔

Reply:
I will decrease the inset to 0.25rem, otherwise in our new input fields it will look terrible
before:

https://github.com/Lundalogik/lime-elements/assets/50618208/f35f9ec7-2e1b-4230-b3c4-c7769d47aaa0


after:


https://github.com/Lundalogik/lime-elements/assets/50618208/f8aac8a5-0269-4d51-b987-688238a2b6af

Reply:
It's a good size, I checked before. I think, we should have it this:  

https://github.com/Lundalogik/lime-elements/assets/50618208/30ddbfb1-b98e-4db1-9def-5bd9d4b0538f